dotnet-engine/Yggdrasil.Engine/Yggdrasil.Engine.csproj

```diff
@@ -12,6 +13,10 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="System.Text.Json" Version="8.0.3" />
```

Comment:
.NET Standard 2.0 does not include `System.Text.Json` by default, so it needs to be added. This is where we can decide which JSON serializer we want to use and stick with it.

I defaulted to `System.Text.Json` for now since that's what we were using, but we can change this to [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/) later if we prefer.